Apps/Inc/Tasks.h

```diff
@@ -180,7 +180,10 @@ void throwTaskError(error_code_t errorCode, callback_t errorCallback, error_sche
 void _assertOSError (OS_ERR err); //TODO: This should be changed to enforce only enum usage
 
 #if DEBUG == 1
-#define assertOSError(err) 
+#define assertOSError(err) \
+        if (err != OS_ERR_NONE) { \
+            printf("Error asserted at %s, line %d: %d\n\r", __FILE__, __LINE__, err); \
+        } \
         _assertOSError(err);
 #else
 #define assertOSError(err) _assertOSError(err);
```

Comment:
WOULDN'T IT MAKE MORE SENSE TO HAVE THIS IN THE _ASSWERTOSFUNCTION ????

Reply:
Can't be in the assert OS function bc otherwise __FILE__ and __LINE__ would just show Tasks.h and line 185.

Reply:
bruh